src/config/index.js

```diff
@@ -4,6 +4,7 @@ export const isStaging = process.env.REACT_APP_STACK === 'STAGING';
 export const isProd = process.env.REACT_APP_STACK === 'PRODUCTION';
 export const isLocal = process.env.NODE_ENV === 'development';
 export const stack = process.env.REACT_APP_STACK;
+export const isOsmTeamsEnabled = false;
```

Comment:
This feature flag is off by default. Setting it to `true` will have OSM Cha start pulling from the [OSM Teams api](https://dev.mapping.team/api/docs), instead of the [mapping-teams endpoint](https://osmcha.mapbox.com/api-docs/#tag/mapping-team)